torchprime/hf_models/configs/default.yaml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  - _self_
+
+# extra env variables you want to pass to the training script
+env:
+  - NUM_SLICE: 1
+  - XLA_PERSISTENT_CACHE_PATH: "/tmp/xla_cache"
+  - PROFILE_LOGDIR: "gs://torchprime/profile"
```

Comment:
See my comment at [1], we don't need to separately configure the profile dir here.

[1]: https://github.com/AI-Hypercomputer/torchprime/pull/95#discussion_r1947049868